src/index.html

```diff
@@ -6,9 +6,60 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo-link" href="#">
+        <img class="logo"
+        src="images/logo.png"
+        alt=" ">
```

Comment:
```suggestion
        <img
          class="logo"
          src="images/logo.png"
          alt="Moyo logo"
        >
```

Reply:
set value for alt-attribute
fix indentations, if you started moving attributes - move every to new line with +2 spaces